cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -385,13 +385,13 @@ def check_index_page(self, separate_archived_courses, org):
 
     @ddt.data(
         # Staff user has course staff access
-        (True, 'staff', None, 3, 18),
-        (False, 'staff', None, 3, 18),
+        (True, 'staff', None, 3, 20),
```

Comment:
@gabor-boros updating query count here as with the use of `has.perm()` we have two additional queries for permission checks.